### PR TITLE
xp readme

### PR DESCRIPTION
**Pull request by Hibiscusofxp**
# xp awesome readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Pintos
```
Comment (zhongming1993) [inline README.md:1]: awesome!

**Review by howryu — COMMENTED** [review]
saca

**Comment by Hibiscusofxp** [comment]
# saca

**Comment by Hibiscusofxp** [comment]
- [x] wei ge is saca
- [ ] wei ge is not saca

**Pull request merged**